src/context/WalletProvider/KeepKeyProvider.tsx

```diff
@@ -48,6 +55,21 @@ const KeepKeyContext = createContext<IKeepKeyContext | null>(null)
 
 export const KeepKeyProvider = ({ children }: { children: React.ReactNode }): JSX.Element => {
   const [state, dispatch] = useReducer(reducer, initialState)
+  const { state: walletState, dispatch: walletDispatch, load } = useWallet()
+  const { wallet } = walletState
+  const keepKeyWallet = isKeepKeyWallet(wallet) ? wallet : undefined
```

Comment:
Can we do this check only twice? e.g once when there is no `wallet` defined yet, a second time after `wallet` is defined. `useMemo()` would allow us to compute the keepKeyWallet variable in a memoized fashion if we use `wallet` as a dependency

Reply:
Very nice, ser: https://github.com/shapeshift/web/pull/1341/commits/b35046c7403c58ee9436cda3e8c8d9be2a688ba7